packages/components/RadioGroup/src/RadioButton.tsx

```diff
@@ -22,11 +22,21 @@ export const RadioButton = compose<IRadioButtonType>({
     // Grabs the context information from RadioGroup (currently selected button and client's onChange callback)
     const info = React.useContext(RadioGroupContext);
 
+    /* We don't want to call the user's onChange multiple times on the same selection. */
+    const changeSelection = () => {
+      if(buttonKey != info.selectedKey)
+        info.onChange && info.onChange(buttonKey);
+    };
+
+    /* We're changing selection on both onPress and onFocus to integrate better with NetUI. For example, for TaskPanes, if focus isn't already landed
```

Comment:
I would avoid references to underlying platform details (NetUI), I would just describe it as radio button changes selection when focus is moved between radio button items as well as on click

Reply:
Makes sense, changing in newest commit